programs/ssl/dtls_client.c

```diff
@@ -99,6 +102,14 @@ int main( int argc, char *argv[] )
     ((void) argc);
     ((void) argv);
 
+#if defined(MBEDTLS_PLATFORM_C)
+    if( ( ret = mbedtls_platform_setup( &platform_ctx ) ) != 0 )
+    {
+        mbedtls_printf( " failed\n  ! mbedtls_platform_setup returned %d\n\n", -ret );
```

Comment:
We're displaying a decimal return value here, yet at line 293, are displaying it as a hexadecimal. We should be self-consistent within the same program.